_providers/gmail.md

```diff
@@ -18,16 +18,18 @@ server:
 opt:
   delete_to_trash: true
 before_login_hint: |
-  For Gmail accounts, you need to create an app-password if you have "2-Step Verification" enabled. If this setting is not available, you need to enable "less secure apps".
-last_checked: 2021-08
+  For Gmail accounts, you need to create an app-password and have "2-Step Verification" enabled.
```

Comment:
Better to flip this. Afair it's impossible to create an app password w/o having 2-Step Verification already enabled

Reply:
makes sense, i added a commit